content/docs/command-reference/add.md

```diff
@@ -341,19 +344,18 @@ Only the hash values of the `dir/` directory (with `.dir` file extension) and
 ## Example: Transfer to the cache
 
 When you have a large dataset in an external location, you may want to add it to
-the <abbr>project</abbr> without having to copy it into the
-<abbr>workspace</abbr>. Maybe your local disk doesn't have enough space, but you
-have set up an
-[external cache](/doc/user-guide/managing-external-data#setting-up-an-external-cache)
```

Comment:
Any particular reason to reword this and move the basic use case to the bottom of the explanation? For me, the old way was easier to understand since it first explains that you lack space locally but have an external cache. Adding directly to cache doesn't have obvious benefit if the cache is local.

Reply:
@dberenbaum I gave this some thought and realized that adding "without having to copy it into the workspace" isn't the base case. The base case is simply to add data that is currently external. `dvc add` doesn't allow this unless you use `--external` (tracks it in place) or (in this case) specify an `--out path` inside the project.

Reply:
I clarified this further now though. The part about not having enough space locally (but an external cache) was missing to complete the motivation for using `--out` vs. just `cp` + `add`. PTAL!

Reply:
p.s. hmmm I thought this was going to be much clearer but I see I'm going in a circle here, a little bit.

The logic behind this case is not simple... I rewrote it completely again 😅 , PTAL -- I think it's better explained now.

Reply:
Much clearer now!